### PR TITLE
Add consistency to page blocks in the BO

### LINKED ISSUE #23015: Size of column in BO is not the same in all page
#### Describe the bug

Size of column in BO is not the same in all page

#### Steps to Reproduce

1. Go to admin -> preference general & order settings 
2. See error

**Screenshots**

Capture d’écran 2021-01-27 à 14 53 40

Capture d’écran 2021-01-27 à 14 53 49

#### Additional information
* PrestaShop version: develop
* PHP version: 7.1

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Every pages block doesn't have the same width, which create inconstitency
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #23015.
| How to test?      | You should introspect migrated pages, See the screen to see a list of modified pages
| Possible impacts? | Every pages block sizes

image

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23020)

### CODE REVIEW

**Comment by NeOMakinG** [comment]
@Julievrz I think wording is fine, I didn't change anything to wordings

**Review by Progi1984 — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig [hunk 1]
```diff
@@ -27,30 +27,32 @@
 {% form_theme handlingForm 'PrestaShopBundle:Admin/TwigTemplateForm:prestashop_ui_kit.html.twig' %}
 
 {% block shipping_preferences_handling %}
-  <div class="col-xl-10">
-    <div id="handling" class="card">
-      <h3 class="card-header">
-        <i class="material-icons">filter_none</i> {{ 'Handling'|trans }}
+  <div class="row justify-content-center">
```
Comment (matks) [inline src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig:30]: Little BC break as the DOM structure changes. Shall we mark it in the PR ?

Comment (matks) [reply src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig:30]: I see actually you have multiple places where you either add or remove
```
<div class="row justify-content-center">
```

So it's multiple small BC breaks ... unless it's bug fixing ?

**Review by matks — COMMENTED** [review]
Code seems OK for me but I'd like to understand the intent behind and the possible BC break impact before approving 😄

File: src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig (hunk 1 above)
Comment (NeOMakinG) [reply src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig:30]: Well, it's bug fixing at it makes the block size smaller with past structure, but you're right it's DOM structure changes and could be BC Breaks but it should be acceptable as you should never select a row or an helper class with JS!

Comment (matks) [reply src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig:30]: > as you should never select a row or an helper class with JS!

I trust you on this one as you're a way better JavaScript developer than I am 😉

**Review by matks — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig (hunk 1 above)
Comment (NeOMakinG) [reply src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/Blocks/shipping_preferences_handling.html.twig:30]: It's not about being better, you're taking risk by selecting rows or bootstrap helper classes, except if you're forced to!

**Comment by sarahdib** [comment]
@NeOMakinG missing the maintenance page 
Capture d’écran 2021-02-22 à 18 16 30

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by aDamelincourt** [comment]
@NeOMakinG Missing this one 
bug_Screenshot 2021-03-03_11-13-40-176
Perhaps this one ?
bug_Screenshot 2021-03-03_11-23-08-848

**Comment by NeOMakinG** [comment]
@aDamelincourt these one can't be done

The first one on the Positions page got a block on the right when you select some modules

The second one is containing multiple blocks if you add some

**Comment by aDamelincourt** [comment]
Validated

**Comment by PierreRambaud** [comment]
Thank you @NeOMakinG

**Pull request merged**